modules/command/src/main/java/org/apache/fluo/command/FluoScan.java

```diff
@@ -97,14 +157,42 @@ public static void main(String[] args) {
     ScanOptions options = ScanOptions.parse(args);
     FluoConfiguration config = CommandUtil.resolveFluoConfig();
     config.setApplicationName(options.getApplicationName());
-    options.overrideFluoConfig(config);
+    // options.overrideFluoConfig(config);
```

Comment:
Why is this commented out?

Reply:
Because in that place it was called before "FluoAdminImpl.mergeZookeeperConfig(config);" that I needed when I call "ScanUtil.scanFluo(...)".
But how I'll remove the properties from the configuration file I'll commented this out.